internal/pkg/manifest/errors.go

```diff
@@ -98,6 +98,29 @@ func (e *errFieldMutualExclusive) Error() string {
 	return fmt.Sprintf(`must specify one, not both, of "%s" and "%s"`, e.firstField, e.secondField)
 }
 
+type errSpecifiedBothIngressFields struct {
+	firstField  string
+	secondField string
+}
+
+func (e *errSpecifiedBothIngressFields) Error() string {
+	return fmt.Sprintf(`must specify one, not both, of "%s" and "%s"`, e.firstField, e.secondField)
+}
+
+func (e *errSpecifiedBothIngressFields) RecommendActions() string {
+	privateOrPublicField := strings.Split(e.firstField, ".")[0]
+	return fmt.Sprintf(`
+It looks like you have specified mutually exclusive fields for Load Balancer configuration.
+The "%s.http.security_group.ingress" has been deprecated and we encourage you to use "%s.http.ingress".
+Following is the example manifest fields that we recommend you to use for your environment.
+
+http:
+  %s:
+    ingress:
+      cdn: true
+`, privateOrPublicField, privateOrPublicField, privateOrPublicField)
```

Comment:
What do you think of a more customized recommended actions?

Something I can think of is we can probably detect what they have in `http.public.security_groups.ingress` and just tell them how to migrate to `http.public.ingress` exactly (instead of giving a general example). 

For example, if we see `http.public.security_groups.ingress.cdn: true` and `http.public.ingress.anewfield: ture` , we could probably just straight up produce the following snippet:
```yaml
http:
  public:
     ingress:
        cdn: true # Migrated from http.public.security_groups.ingress.cdn.
        anewfield: true
```

Reply:
As per the discussion keeping the same error message.